test/authenticator.spec.ts

```diff
@@ -166,12 +152,10 @@ describe('Decentraland Crypto', function () {
       const result = await Authenticator.validateSignature(
         'QmWyFNeHbxXaPtUnzKvDZPpKSa4d5anZEZEFJ8TC1WgcfU',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/2c902c2e3b8947d3b34bba7ca48635fc'
-        )
+        new HTTPProvider('https://rpc.decentraland.org/mainnet')
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?